package.json

```diff
@@ -84,6 +84,7 @@
     "@elastic/search-ui-site-search-connector": "1.7.0",
     "@mapbox/mapbox-gl-supported": "^2.0.1",
     "@mapbox/mr-ui": "^2.7.0",
+    "@mapbox/remark-lint-mapbox": "^4.1.0",
```

Comment:
Does this need to be explicitly added here?

Reply:
No, I believe this was left over from my forced update to `remark-lint-mapbox` in order to have access to the new content type.  This packages is already a dependency of `remark-config-docs` but rather than wait for dependabot to push an updated release (24 hrs?) I ran `npm i @mapbox/remark-lint-mapbox@latest --force` and I believe that's what added this. I was following the [release instructions here](https://github.com/mapbox/remark-lint-mapbox#how-to-release). 

I've removed this in the my most recent commit and `remark-config-docs` should have the most recent version (if the former release process is still working)